rclcpp/include/rclcpp/node.hpp

```diff
@@ -663,6 +663,16 @@ class Node : public std::enable_shared_from_this<Node>
   const NodeOptions &
   get_node_options() const;
 
+  /// Manually assert that this Node is alive (for RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE)
+  /**
+   * If the rmw Liveliness policy is set to RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE, the creator
+   * of this node may manually call `assert_liveliness` at some point in time to signal to the rest
+   * of the system that this Node is still alive.
```

Comment:
The return type is neither documented nor obvious (to me). Does this ever throw?

Reply:
Will add documentation of the return type. The documentation will not have `\throws` indicator, because the function will not throw.